linkerd/app/inbound/src/http/mod.rs

```diff
@@ -1,256 +1,34 @@
-                    AllowProfile(config.allow_discovery.clone()),
-                ))
-                .instrument(|_: &Target| debug_span!("profile"))
-                .push_on_response(
-                    svc::layers()
-                        .push(http::BoxResponse::layer())
-                        .push(svc::layer::mk(svc::SpawnReady::new)),
-                )
-                // Skip the profile stack if it takes too long to become ready.
-                .push_when_unready(
-                    config.profile_idle_timeout,
-                    target
-                        .clone()
-                        .push_on_response(svc::layer::mk(svc::SpawnReady::new))
-                        .into_inner(),
-                )
-                .check_new_service::<Target, http::Request<BoxBody>>()
-                .push_on_response(
-                    svc::layers()
-                        .push(
-                            rt.metrics
-                                .stack
-                                .layer(crate::stack_labels("http", "logical")),
-                        )
-                        .push(svc::FailFast::layer(
-                            "HTTP Logical",
-                            config.proxy.dispatch_timeout,
-                        ))
-                        .push_spawn_buffer(config.proxy.buffer_capacity),
-                )
-                .push_cache(config.proxy.cache_max_idle_age)
-                .push_on_response(
-                    svc::layers()
-                        .push(http::Retain::layer())
-                        .push(http::BoxResponse::layer()),
-                )
-                .check_new_service::<Target, http::Request<http::BoxBody>>()
-                // Routes each request to a target, obtains a service for that
-                // target, and dispatches the request.
-                .instrument_from_target()
-                .push(svc::BoxNewService::layer())
-                .push(svc::NewRouter::layer(RequestTarget::from))
-                // Used by tap.
-                .push_http_insert_target::<HttpAccept>()
-                .push(svc::BoxNewService::layer())
-        })
-    }
-}
-
 fn trace_labels() -> std::collections::HashMap<String, String> {
     let mut l = std::collections::HashMap::new();
     l.insert("direction".to_string(), "inbound".to_string());
     l
 }
 
 #[cfg(fuzzing)]
-pub mod fuzz_logic {
+pub mod fuzz {
     use crate::{
-        target::{HttpAccept, TcpAccept, TcpEndpoint},
+        http::router::HttpEndpoint,
         test_util::{
             support::{connect::Connect, http_util, profile, resolver},
             *,
         },
         Config, Inbound,
     };
-    use hyper::http;
     use hyper::{client::conn::Builder as ClientBuilder, Body, Request, Response};
     use libfuzzer_sys::arbitrary::Arbitrary;
     use linkerd_app_core::{
-        io, proxy,
+        identity, io,
+        proxy::http,
         svc::{self, NewService, Param},
```

Comment:
`Param` import is unused
```suggestion
        svc::{self, NewService},
```